cms/djangoapps/contentstore/views/course.py

```diff
@@ -899,8 +899,7 @@ def settings_handler(request, course_key_string):
                     ee_min_score_pct = request.json.get('entrance_exam_minimum_score_pct', None)
 
                     # If the entrance exam box on the settings screen has been checked,
-                    # and the course does not already have an entrance exam attached...
-                    if entrance_exam_enabled and not course_entrance_exam_present:
+                    if entrance_exam_enabled:
                         # Load the default minimum score threshold from settings, then try to override it
```

Comment:
Hmmm, matt i think i should revert back this condition into its original state like as >  
  if entrance_exam_enabled and not course_entrance_exam_present:
Because during creating an entrance exam , we are first cleaning out any old entrance exams in entrance_exam.py line 84.
So this condition  > if entrance_exam_enabled: -- will create a new instance of an entrance exam instead of updating the minimum passing score of a pre created exam.